src/screens/Accounts/index.js

```diff
@@ -51,6 +51,27 @@ class Accounts extends NavigatorComponent<Props & Actions & AccountsState> {
     });
   }
 
+  static showResetPasscodeSuccess = async (navigator: Navigator) => {
```

Comment:
Please show an alert to tell the user that password was successfully reset.

Reply:
done